build.sbt

```diff
@@ -36,7 +36,7 @@ sourceGenerators in Compile ++= Seq(
 )
 
 // Lib dependencies
-val sonarVersion = "6.7"
+val sonarVersion = "7.3"
 libraryDependencies ++= List(
```

Comment:
I remember, I once said that when we refactored the old scalastyle module, there will be some dependencies we could remove... I think it was `sonar-core` and `guava`, but I'm not sure. I will try to search for that discussion after the review. But if I couldn't find it, could you please try removing some dependencies and check if we don't need?